public/content/de/basics/basic-types.md

```diff
@@ -0,0 +1,118 @@
+# Elementare Datentypen
+
+D bietet eine Vielzahl an elementaren Datentypen, die immer die gleiche Größe haben -
+**unabhängig** von der Platform.
+Die einzige Ausnahme ist der Typ `real` für welchen der Kompiler
+die höchstmöglichste Genauigkeit für Gleitkommazahlen wählt.
+Es besteht kein Unterschied zwischen der Große einer Ganzzahl - unabhängig davon
```

Comment:
yeah thats better. Just `Ganzzahltypen` then. Try to not mix up types and typen in the file. Either one or the other one but not both